packages/language-server/src/completion/completions.json

```diff
@@ -40,7 +40,7 @@
       "label": "Unsupported",
       "insertText": "Unsupported(\"$0\")",
       "documentation": "An arbitrary database column type, for which Prisma has no syntax. Fields of type `Unsupported` work with Prisma Migrate and introspection, but are not exposed in Prisma Client.",
-      "fullSignature": "Unsupported(_ name: String)",
+      "fullSignature": "Unsupported(name?: String)",
```

Comment:
I think the `_` is a notation we also use in our documentation: https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#signature-5 It is unfortunately not used consistently. I started a thread slack about it.

In docs we also use the `?` on the type, not the argument usually.

Reply:
Indeed, it probably started with docs doing a copy and paste of this and evolved separately at some point

Reply:
No. I think we can ignore that aspect (which one of the fields can be used without a name) for now and just make the rest consistent.

Reply:
No idea, I tried to clean this up, but completely abandoned this after it was blocked, to focus on other priorities.